other/test-setup/vitejs_plugin_react.cjs

```diff
@@ -0,0 +1,2 @@
+// react types are missing when import as default to ESM module
```

Comment:
Let's rename this file to `vitejs-plugin-react.cjs`

Reply:
done